### ISSUE TITLE
Warning issue: couldn't connect

### ISSUE THREAD

**Issue by Wisnuuh** [midori-browser/core#466]
This is what happen

```
** (midori:35218): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /run/user/1000/at-spi/bus: Permission denied
Segmentation fault (core dumped)
```

**Comment by Psycomentis06** [comment]
This warning comes from the GTK runtime and it's not something that break the app. And this version of midori is deprecated maybe your GTK version is not compatible try the new electron based midori that can be found [here](https://astian.org/midori-browser/download/)

**Comment by Ponchale** [comment]
The version based on WebKit is not recommended for download, it is recommended to migrate to version 11 and the new maintenance cycle, where the same lightness, privacy and security and many more are still maintained.

https://astian.org/midori-browser
